packages/charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -227,6 +269,7 @@ export class LineChartBase extends React.Component<ILineChartProps, ILineChartSt
         const y2 = this._points[i].data[j].y;
         const xAxisCalloutData = this._points[i].data[j - 1].xAxisCalloutData;
         if (this.state.activeLegend === legendVal || this.state.activeLegend === '') {
+          /* eslint-disable react/jsx-no-bind */
```

Comment:
This shouldn't be needed--the lint rule was disabled recently for built-in elements